### PR TITLE
Added code to functions in household.py

### PR DESCRIPTION
**Pull request by joel-becker**
- [x] Get c1
- [x] Get c2
- [x] Write form of marginal utility

### CODE REVIEW

**Comment by rickecon** [comment]
@joel-becker Please revert those changes to the `README.md` file. Thank you so much.

**Comment by EdJeeOnGitHub** [comment]
@rickecon  This is a hill we're prepared to die on. Group 2 micdrop out

File: Projects/Project2/household.py [hunk 1]
```diff
@@ -33,7 +35,11 @@ def get_c2(b2, args):
     all the arguments necessary in equation (27) as well as whatever
     arguments are necessary in FirmsMC.get_r() and FirmsMC.get_w().
     '''
-    # Put code here.
+    # unpack arguments
+    nvec, alpha, A, delta = args
+
+    # specify c2
+    c2 = (1+ FirmsMC.get_r(b2, (nvec, alpha, A)))*b2 + FirmMC.get_w(b2, (nvec, alpha, A, delta))*nvec[1]
```
Comment (EdJeeOnGitHub) [inline Projects/Project2/household.py:43]: c2 is here I think @rickecon

**Comment by rickecon** [comment]
@joel-becker And you forgot to submit `get_c2()`. Please add.

**Comment by rickecon** [comment]
In line 23 of `household.py`, unpack `n1` inside of the function rather than passing it in separately from `nvec`. The code inside the function should be something like:
```python
n1, n2 = nvec
```

**Comment by rickecon** [comment]
`c2` is definitely there. In line 43, split the pieces out. For example
```python
r = FirmsMC.get_r(...)
w = FirmsMC.get_w(...)
```

**Pull request merged**